.github/workflows/errno.yml

```diff
@@ -0,0 +1,32 @@
+name: Error numbers
+on:
+  pull_request:
+    paths:
+      - 'lib/libc/minimal/include/errno.h'
+
+jobs:
+  check-errno:
+    runs-on: ubuntu-latest
+    container:
+      image: zephyrprojectrtos/ci:v0.17.1
+      options: '--entrypoint /bin/bash'
```

Comment:
isn't zephyr CI entry point already bash?

Reply:
I don't think so. Only the [user image](https://github.com/zephyrproject-rtos/docker-image/blob/master/Dockerfile.user#L20) seems to set an entry point

Reply:
example-application doesn't need that, and uses the same container: https://github.com/teslabs/example-application/blob/main/.github/workflows/build.yml